docs/interpreter/spark.md

```diff
@@ -255,3 +255,29 @@ select * from ${table=defaultTableName} where text like '%${search}%'
 ```
 
 To learn more about dynamic form, checkout [Dynamic Form](../manual/dynamicform.html).
+
+<br />
+### Setting up Zeppelin with Kerberos
+<hr />
```

Comment:
@sejunra After applying this PR, I noticed there is already horizon line below the title because of Zeppelin's `style.css`. 

![screen shot 2016-02-08 at 6 27 32 pm](https://cloud.githubusercontent.com/assets/10060731/12882003/a6110c04-ce91-11e5-9a6e-59257ee0828e.png)

So, therefore you don't need to add additional <hr> tag : )